### PR TITLE
Add first version of the registry

### PR DESCRIPTION
**Pull request by stefanjudis**
First version of the registry. ;)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        url: 'https://github.com/gatsbymanor/gatsby-theme-massively'
+      }
+    }
+  },
+  {
+    name: 'lens',
+    description: '...',
+    contentSources: {
+      plain: {
+        url: 'https://github.com/gatsbymanor/gatsby-lens'
```
Comment (nodox) [inline index.js]: Typo: `theme` is missing from the name.

Should be `https://github.com/gatsbymanor/gatsby-theme-lens`

**Review by nodox — COMMENTED** [review]
Looks good! Small typo.

General thoughts:
I think the structure should be
```
  {
    name: 'massively',
    description: '...',
    url: 'https://github.com/gatsbymanor/gatsby-theme-massively',
    contentSources: {
      plain: {
         ... // Not sure what goes here
      },
      contentful: {
        ... // Not sure 
      }
    }
  },
```
The url being separated from the `contentSources`.

The current structure implies that for one theme there are X number of themes with the same style designs but with a different data sources. There could be a discrepancy between the designs found at each of those endpoints.

If you think of `jay` as a bootstrapping tool then when you specify a data source it should make the necessary modifications to that theme to enable it with that data source. 

If we continue with the idea that Jay wraps gatsby and adds functionality, maybe it install plugins, and gives the components the necessary information from a config file. The components will then read that data based on what you gave it from the cli. 

Maybe it performs all the necessary adjustments that are needed to make those sources work with the theme (similar to a plugins). 

Not sure exactly. What do you think?

**Comment by stefanjudis** [comment]
You're absolutely right. This is way better. I wanted to go the easy route. ;) Fixed. :)

Ach... dropped the Urls.. One sec. :D

**Comment by stefanjudis** [comment]
Fixed. :)

**Review by nodox — APPROVED** [review]
This looks great!

**Comment by stefanjudis** [comment]
When time could you push the publish button here? 😊
(I mean publish it to npm) :D

**Comment by nodox** [comment]
Registry published!

**Pull request merged**